backend/common/utils/cxg_generation_utils.py

```diff
@@ -52,7 +80,12 @@ def convert_dataframe_to_cxg_array(cxg_container, dataframe_name, dataframe, ind
     tdb_attrs = []
 
     for column_name, column_values in dataframe.items():
-        dtype, hints = get_dtype_and_schema_of_array(column_values)
+        # Cast 'in_tissue' column values as boolean to make it categorical
+        # https://github.com/chanzuckerberg/single-cell-explorer/issues/841
+        if column_name == "in_tissue":
+            dtype, hints = get_dtype_and_schema_of_array(column_values.astype(bool))
```

Comment:
this is to treat the [reported issue](https://github.com/chanzuckerberg/single-cell-explorer/issues/841) for `in_tissue` boolean column that sometimes stored as int. After casting it as bool it will appear as categorical in explorer